src/tab/Tab.ts

```diff
@@ -0,0 +1,72 @@
+import { css, html, TemplateResult } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { OmniElement } from '../core/OmniElement.js';
+
+/**
+ * Control that can be used to display slotted content, for use within an omni-tabs component.
+ *
+ * @import
+ * ```js
+ * import '@capitec/omni-components/tab';
+ * ```
+ * @example
+ * html```
+ * <omni-tab header='Tab 1'>
+ *  <span>Tab 1 Content</span>
+ * </omni-tab>
+ * ```
+ *
+ * @element omni-tab
+ *
+ * @slot - Content to render inside the component body.
+ *
+ */
+@customElement('omni-tab')
+export class Tab extends OmniElement {
+    /**
+     * Tab header label.
```

Comment:
Maybe mention here the option to use the header component as well for more complex header layouts.

Reply:
Updated